CONTRIBUTING.rst

```diff
@@ -62,6 +62,11 @@ To set up `import-linter` for local development:
 
     tox
 
+   Or if you're a `uv <https://docs.astral.sh/uv/>` user and want to just run against eg. Python 3.12::
```

Comment:
I've just merged a completely new approach which prescribes uv, so we should probably drop this commit.